exercicios/hello-world.csv

```diff
@@ -0,0 +1 @@
+"Imprima Hello World! na tela","","Hello World!"
```

Comment:
Podia ser "imprima Hello World na tela se for sem input, ou Hello Pedro!, se o input for Pedro